app/models.py

```diff
@@ -260,6 +260,8 @@ class SupplierFramework(db.Model):
                              db.ForeignKey('frameworks.id'),
                              primary_key=True)
     declaration = db.Column(JSON)
+    on_framework = db.Column(db.Boolean, nullable=True)
```

Comment:
So this is another 3-way boolean, right?

If a given supplier:

- applies and succeeds -> `on_framework = True`
- applies and fails -> `on_framework = False`
- doesn't apply -> no `SupplierFramework` record

Have I got that right?